packages/devcmd/src/utils/npm_utils.ts

```diff
@@ -0,0 +1,11 @@
+import { spawn } from "child_process";
+
+export async function checkPackageAvailable(packageName: string, directory: string) {
```

Comment:
as we recently discussed, since this function is exported, it should have a full signature, i.e. its return type `Promise<boolean>` should be explicitly defined :)